core/src/test/java/org/apache/iceberg/mapping/TestMappingUpdates.java

```diff
@@ -216,34 +216,4 @@ public void testRenameComplexFieldMappingUpdate() {
             ))),
         pointUpdated.asMappedFields());
   }
-
-  @Test
-  public void testMappingUpdateFailureSkipsMappingUpdate() {
```

Comment:
This test is no longer valid because there change that caused an exception while updating the mapping is now supported. Since there aren't changes that cause a failure and there isn't a good way to inject an implementation that fails, I'm just removing this test.